pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -55,6 +55,26 @@
 , includeFortifyHeaders ? null
 }:
 
+let libcxx_args = libcxx; in
```

Comment:
Instead of the sequence of `let`-expressions one could also add `@args` after the lambda's formal parameters, and then define `libcxx = if args.libcxx != null then args.libcxx else ...`. Can't say which one is less "wacky". I've seen (and committed) both in nixpkgs